File: api/code-samples/python/eth/transaction/getTransactionByHash.py
import json
from opensdk.sdk import OpenSDK

host = "https://api.baobab.klaytn.net:8651"

blockHash = "0xaca5d9a1ed8b86b1ef61431b2bedfc99a66eaefc3a7e1cffdf9ff53653956a67"

sdk = OpenSDK(host)
eth_response = sdk.eth.get_transaction_by_hash(blockHash)

print(json.loads(eth_response.response.data))

File: web3rpc/rpc-specs/code-samples/javascript/eth/transaction/getTransactionByHash.js
const { JsonRpcProvider } = require("@klaytn/ethers-ext");
(() => {
  const provider = new JsonRpcProvider("https://public-en-baobab.klaytn.net");
  const transactionHash =
    "0xc6acc62baaa57483da8d5e08aaed1907d82f0e25bd553ce3745ef1bc7b7f4476";
  provider.eth
    .getTransactionByHash(transactionHash, {}, (err, data, response) => {})
    .then((data) => {
      console.log(data);
    });
})();
